.circleci/config.yml

```diff
@@ -7,14 +7,14 @@ aliases:
     restore_cache:
       name: "Restoring yarn cache"
       keys:
-        - v2-yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
-        - v2-yarn-cache-{{ .Branch }}
-        - v2-yarn-cache
+        - v3-yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
What happened? 🤔

Reply:
Assumed CI‘s cache got choked. It is an odd react-intl issue. Will check later.